src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2292,8 +2292,12 @@ protected virtual LocalFunctionSymbol LookupLocalFunction(SyntaxToken nameToken)
             return Next.LookupLocalFunction(nameToken);
         }
 
+        protected virtual ImmutableArray<MethodSymbol> MethodSymbolsWithYield =>
```

Comment:
>MethodSymbolsWithYield [](start = 55, length = 22)

It looks like this property deserves a good doc comment. Also, we usually define APIs like this in Binder.cs.